mods/pvp/medkits/init.lua

```diff
@@ -130,9 +130,12 @@ end)
 -- If player takes damage while healing,
 -- stop regen and revert back to original state
 minetest.register_on_player_hpchange(function(player, hp, reason)
+	local name = player:get_player_name()
 	if hp < 0 then
-		if players[player:get_player_name()] then
-			stop_healing(player, "damage")
+		if players[name] then
+			players[name] = nil -- Don't use stop_healing(), it uses set_hp() and won't allocate deaths or score properly
+			minetest.chat_send_player(name, minetest.colorize("#FF4444", "Your healing was interrupted"..reason_handler("damage")))
```

Comment:
```suggestion
			interrupted = "damage"
```
This could work better, but no idea if it works

Reply:
Since stop_healing() isn't called, it would just be setting the variable value without accomplishing anything, I do think I should remove the line though, doesn't seem very useful